src/components/Users/UsersContainer.js

```diff
@@ -63,7 +63,7 @@ UsersContainer.defaultProps = {
   currentPage: PropTypes.number,
   totalUsersCount: PropTypes.number,
   isFetching: PropTypes.bool,
-  followingInProgress: PropTypes.bool,
+  followingInProgress: PropTypes.array,
```

Comment:
See the comments above.

Reply:
I would prefer to do this upon completion of priority tasks